Formula/streamlink.rb

```diff
@@ -85,12 +85,11 @@ class Streamlink < Formula
   end
 
   def install
-    virtualenv_install_with_resources
-    man1.install_symlink libexec/"share/man/man1/streamlink.1"
+    virtualenv_install_with_resources(link_manpages: true)
   end
 
   test do
-    system "#{bin}/streamlink", "https://vimeo.com/189776460", "360p", "-o", "video.mp4"
+    system "#{bin}/streamlink", "https://youtu.be/he2a4xK8ctk", "360p", "-o", "video.mp4"
     assert_match "video.mp4: ISO Media, MP4 v2", shell_output("file video.mp4")
     output = shell_output("#{bin}/streamlink -l debug 'https://ok.ru/video/3388934659879'")
```

Comment:
So does look like `vimeo` is blocked on Google Cloud self-hosted runner. Youtube seems to work. The most used test video is too long, so just grabbed the short test video in `you-get`.

Sadly, `ok.ru` may also be blocked. May keep on macOS (to test a different source) and use a different Youtube video on Linux.